documentation/src/main/asciidoc/topics/proc_configuring_file_stores.adoc

```diff
@@ -2,21 +2,26 @@
 = Configuring file-based cache stores
 Add file-based cache stores to {brandname} to persist data on the host filesystem.
 
+[IMPORTANT]
+====
+To retain data during clustered cache restarts, use a shared store. 
+See, link:https://access.redhat.com/solutions/6960765[Adding persistence in a cache in RHDG 8 in OCP 4] ({RedHat} Customer Portal). 
+====
+
 .Prerequisites
 
 * Enable global state and configure a global persistent location if you are configuring embedded caches.
 
 .Procedure
 
 . Add the `persistence` element to your cache configuration.
+
 . Optionally specify `true` as the value for the `passivation` attribute to write to the file-based cache store only when data is evicted from memory.
+
 . Include the `file-store` element and configure attributes as appropriate.
-. Specify `false` as the value for the `shared` attribute.
-+
-File-based cache stores should always be unique to each {brandname} instance.
-If you want to use the same persistent across a cluster, configure shared storage such as a JDBC string-based cache store .
```

Comment:
Both of these two points are valid. If we're removing the description of the `shared` attribute here, we should move this information to `con_file_based_cache_stores.adoc`.

Reply:
Thanks. Moved sentences to con_file_based_cache_stores.adoc.

Reply:
I can't see that on the diff

Reply:
HI @ryanemerson . I don't know what's happening here, as I see this on the PR:

![ksnip_20221005-111015](https://user-images.githubusercontent.com/57954076/194036749-f74de5f2-cf5d-4c59-83d6-e4b5b0ac4cad.png)

Reply:
I can see it now :+1: